src/Uno.Analyzers.Tests/UnoNotImplementedTests.cs

```diff
@@ -104,28 +89,17 @@ class TypeName
                     {
                         public TypeName()
                         {
-                           var a = new Uno.TestClass();
+                           var a = [|new Uno.TestClass()|];
```

Comment:
How do the new verifiers validate that the diagnostic is the right one ?

Reply:
@jeromelaban The `DiagnosticAnalyzer` specified as type parameter of `CSharpCodeFixVerifier` has a single `DiagnosticDescriptor`, so there aren't multiple options to be reported. In case the analyzer have multiple supported diagnostics, the syntax to use will be `{|ID:code|}`, otherwise the test should fail unless `MarkupOptions = MarkupOptions.UseFirstDescriptor` is specified.